ONSdigital/dp-import
# converted to go module and added Makefile

### What

- Converted to go module (library) and removed vendors folder
- Migrated from `go-ns/avro` to `dp-kafka/v2/avro`
- Added schema for `cantabular dataset instance started` event.

### How to review

- Make sure the new schema makes sense
- Run `make audit` and validate that there is no vulnerability

### Who can review

Anyone
